### DISCUSSION TITLE
bookmark colours

### DISCUSSION THREAD

**Discussion by lvmm** [sumatrapdfreader/sumatrapdf#2335] [Q&A]
Hello,

I tried setting green on black colours for ebooks in the advanced options:

EbookUI [
	FontName = Lucida Sans
	FontSize = 12.5
	TextColor = #008000
	BackgroundColor = #000000
	UseFixedPageUI = false
]

It works fine for the book body but in the contents (bookmarks) pane the focused item is now black on black and the highlight bar is black on very dark blue, both unreadable. Unfortunately bookmarks pane colours are not documented here https://www.sumatrapdfreader.org/settings/settings3-3-1 How can I change them? 3.3.3
![image](https://user-images.githubusercontent.com/11752339/153579184-21c808c1-4983-4dd5-9dfb-3cd3613b7e93.png)

**Comment by GitHubRulesOK** [comment]
Long time know issue with inverted colours each time it has been fixed to correctly to follow paper backgroud colour
since coloured bookmarks are usually black in the PDF etc. and black bookmarks on black background does not work well
unlike black on a white background. see https://github.com/sumatrapdfreader/sumatrapdf/issues/2224#issuecomment-955492820
